assets/app/views/settings.html

```diff
@@ -8,16 +8,16 @@
       <h2>General information</h2>
       <!-- TODO make these fields editable - probably with a cog icon that flips into inline-editing -->
       <dl class="dl-horizontal left">
-        <dt>Name:</dt>
-        <dd>{{projectName}}</dd>
-        <dt>Display name:</dt>
-        <dd ng-if="project | displayName: true">{{project | displayName: true}}</dd>
-        <dd ng-if="!(project | displayName: true)"><em>No display name</em></dd>
-        <dt>Description:</dt>
-        <dd ng-if="project | description">
+        <dt class="name">Name:</dt>
```

Comment:
+1, I'll try use another strategy. Actually an earlier proposal used the indexes of dd/dt inside of the dl, does it sound reasonable?